stbt_control_relay.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+"""
+Allows using any of the stbt remote control backends remotely using the lirc
+protocol.
+
+Presents the same socket protocol as lircd but sending keypresses using any of
+stbt's controls.  This allows for example controlling a roku over its HTTP
+interface from some software that only speaks lirc.
+
+Example usage:
+
+    $ stbt control-relay file:example
+
+Listens on `/var/run/lirc/lircd` for lirc clients.  Keypress sent will be
+written to the file example.  So
+
+    $ irsend SEND_ONCE stbt KEY_UP
+
+Will write the text "KEY_UP" to the file `example`.
+
+    $ stbt control-relay --input=lircd:lircd.sock \\
+          roku:192.168.1.13 samsung:192.168.1.14
+
+Listens on lircd.sock and will forward keypresses to the roku at 192.168.1.13
+using its HTTP protocol and to the Samsung TV at 192.168.1.14 using its TCP
+protocol.  So
+
+    $ irsend -d lircd.sock SEND_ONCE stbt KEY_OK
```

Comment:
Actually it will get ignored by all backends because `fake_lircd_listen` only tells us the key. Each backend's configuration tells it which control to use (which lirc config file if it's a lirc backend, etc).